lib/Serialization/Serialization.h

```diff
@@ -297,6 +297,10 @@ class Serializer : public SerializerBase {
   /// The decls that adopt compiler-known protocols.
   SmallVector<DeclID, 2> KnownProtocolAdopters[NumKnownProtocols];
 
+  Serializer(
+      ArrayRef<unsigned char> signature, ModuleOrSourceFile DC) :
+          SerializerBase(signature, DC) {}
```

Comment:
Is this used by code in this PR?

Reply:
Ah, thanks for pointing this out - it's unused. I was originally adding the flag to Serializer as well, but since moved it to LangOpts. Sorry for missing this one!